src/routes/about/+page.server.ts

```diff
@@ -0,0 +1,34 @@
+import { INSTAGRAM_TOKEN } from '$env/static/private';
+import type { PageServerLoad } from './$types';
+
+const instagramUrl = 'https://graph.instagram.com/me/media?';
+const fields = 'caption,media_url,username,permalink,media_type,thumbnail_url';
+const limit = '6';
+
+export type Post = {
+  caption?: string;
+  media_type: string;
+  thumbnail_url?: string;
+  media_url: string;
+  username: string;
+  permalink: string;
+  id: string;
+}
+
+export const load: PageServerLoad = async () => {
```

Comment:
We should move this to an [API route](https://kit.svelte.dev/docs/routing#server) then call it from the client in an `onMount` function so that it doesn't block the page load.